gneiss/q2/_type.py

```diff
@@ -0,0 +1,23 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+from qiime.plugin import SemanticType
+from q2_types.feature_table import (
+    FeatureTable, Frequency, BIOMV100DirFmt, BIOMV210DirFmt)
+
+
+# Regression types
+Regression = SemanticType('Regression', field_names=['type'])
```

Comment:
Thinking about changing this to `BalanceRegression` to avoid potential name clashing with other Regression types.  For example there are techniques such as Multinomial Regression or Distance based Regression